src/simply/simply_voice.c

```diff
@@ -0,0 +1,130 @@
+#include "simply_voice.h"
+
+#include "simply_msg.h"
+
+#include "simply.h"
+
+#include <pebble.h>
+
+typedef struct VoiceStartPacket VoiceStartPacket;
+
+struct __attribute__((__packed__)) VoiceStartPacket {
+  Packet packet;
+  bool enableConfirmation;
+};
+
+
+typedef struct VoiceDataPacket VoiceDataPacket;
+
+struct __attribute__((__packed__)) VoiceDataPacket {
+  Packet packet;
+  int8_t status;
+  char result[];
+};
+
+static SimplyVoice *s_voice;
+
+static bool send_voice_data(int status, char *transcription) {
+  size_t transcription_length = strlen(transcription);
```

Comment:
Adding +1 resulted in stray characters on the JS side.. without the +1 works as expected